frontend/src/app/features/boards/index-page/boards-index-page.component.ts

```diff
@@ -48,7 +49,18 @@ export class BoardsIndexPageComponent extends UntilDestroyedMixin implements OnI
   public boards$:Observable<Board[]> = this
     .apiV3Service
     .boards
-    .observeAll();
+    .observeAll()
+    .pipe(
+      map((boards:Board[]) => boards.sort((a, b) => {
```

Comment:
There is a function called `localCompare` which does exactly this for you. So you can shorten this whole block to something like this 
```
map((boards:Board[]) => boards.sort((a, b) => a.name.localeCompare(b.name))),
```